instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -180,8 +180,26 @@ main()
     cp -rf ${repoDir}/zm-webclient-portal-example/example ${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/portals
 
     echo "\t\t***** downloads content *****" >> ${buildLogFile}
-    mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/downloads
-    cp -rf ${repoDir}/zm-downloads/. ${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/downloads
+    downloadsDir=${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/downloads
+    mkdir -p ${downloadsDir}
+    cp -rf ${repoDir}/zm-downloads/. ${downloadsDir}
+
+    if [ "${buildType}" == "NETWORK" ]
```

Comment:
Yes, we need to do this thing for all the network related stuff not just for this one but something not targeted now. We will create tasks for the same and will see.